config/locales/views.de.yml

```diff
@@ -777,14 +777,11 @@ de:
         flash:
           privacy_policy_not_accepted: Um die Registrierung abzuschliessen, muss der Datenschutzerklärung zugestimmt werden.
     self_inscription:
-      new:
+      show:
```

Comment:
Ich bin hier hal mit der github suche und dem string `self_inscription` über die organisiation und das scheint nirgends überschrieben zu werden, dh von den wagons her okay, allerdings fallen so die anderen sprachen wieder auf das default de zurück, sprich wir sollten wahrscheinlich den change in den anderen sprachen nachziehen oder wir verwenden die bestehenden keys

Reply:
Hab den change hier nachgezogen bei den anderen übersetzungen.